scripts/cache/copy_cache.py

```diff
@@ -0,0 +1,141 @@
+"""Utility for copying caches from SQLite to MongoDB.
+
+Example usage:
+
+  python3 scripts/cache/copy_cache.py --organization openai \
```

Comment:
To double check, does this command work for organizations that don't exist MongoDB yet?